---
title: Hierarchical graph transformer with contrastive learning for gene regulatory
  network inference
date: '2024-10-14'
linkTitle: https://pubmed.ncbi.nlm.nih.gov/39401117/?utm_source=curl&utm_medium=rss&utm_campaign=pubmed-2&utm_content=1x5bM_TNL8gjogAcnslpo2s2PbDe-61JVM2h9yowOYSiZ7Dkrt&fc=20220919211934&ff=20241015193453&v=2.18.0.post9+e462414
source: (graph neural network) AND (gene)
description: Gene regulatory networks (GRNs) are crucial for understanding gene regulation
  and cellular processes. Inferring GRNs helps uncover regulatory pathways, shedding
  light on the regulation and development of cellular processes. With the rise of
  high-throughput sequencing and advancements in computational technology, computational
  models have emerged as cost-effective alternatives to traditional experimental studies.
  Moreover, the surge in ChIPseq data for TF-DNA binding has catalyzed the ...
disable_comments: true
---
Gene regulatory networks (GRNs) are crucial for understanding gene regulation and cellular processes. Inferring GRNs helps uncover regulatory pathways, shedding light on the regulation and development of cellular processes. With the rise of high-throughput sequencing and advancements in computational technology, computational models have emerged as cost-effective alternatives to traditional experimental studies. Moreover, the surge in ChIPseq data for TF-DNA binding has catalyzed the ...