---
title: 'iGTP: Learning interpretable cellular embedding for inferring biological mechanisms
  underlying single-cell transcriptomics'
date: '2024-12-09'
linkTitle: https://pubmed.ncbi.nlm.nih.gov/39649598/?utm_source=curl&utm_medium=rss&utm_campaign=pubmed-2&utm_content=1x5bM_TNL8gjogAcnslpo2s2PbDe-61JVM2h9yowOYSiZ7Dkrt&fc=20220919211934&ff=20241209184151&v=2.18.0.post9+e462414
source: (graph neural network) AND (gene)
description: Deep-learning models like Variational AutoEncoder have enabled low dimensional
  cellular embedding representation for large-scale single-cell transcriptomes and
  shown great flexibility in downstream tasks. However, biologically meaningful latent
  space is usually missing if no specific structure is designed. Here, we engineered
  a novel interpretable generative transcriptional program (iGTP) framework that could
  model the importance of transcriptional program (TP) space and ...
disable_comments: true
---
Deep-learning models like Variational AutoEncoder have enabled low dimensional cellular embedding representation for large-scale single-cell transcriptomes and shown great flexibility in downstream tasks. However, biologically meaningful latent space is usually missing if no specific structure is designed. Here, we engineered a novel interpretable generative transcriptional program (iGTP) framework that could model the importance of transcriptional program (TP) space and ...