---
title: Improving plant miRNA-target prediction with self-supervised k-mer embedding
  and spectral graph convolutional neural network
date: '2024-05-27'
linkTitle: https://pubmed.ncbi.nlm.nih.gov/38799058/?utm_source=curl&utm_medium=rss&utm_campaign=pubmed-2&utm_content=1x5bM_TNL8gjogAcnslpo2s2PbDe-61JVM2h9yowOYSiZ7Dkrt&fc=20220919211934&ff=20240527181616&v=2.18.0.post9+e462414
source: (graph neural network) AND (gene)
description: Deciphering the targets of microRNAs (miRNAs) in plants is crucial for
  comprehending their function and the variation in phenotype that they cause. As
  the highly cell-specific nature of miRNA regulation, recent computational approaches
  usually utilize expression data to identify the most physiologically relevant targets.
  Although these methods are effective, they typically require a large sample size
  and high-depth sequencing to detect potential miRNA-target pairs, thereby limiting
  ...
disable_comments: true
---
Deciphering the targets of microRNAs (miRNAs) in plants is crucial for comprehending their function and the variation in phenotype that they cause. As the highly cell-specific nature of miRNA regulation, recent computational approaches usually utilize expression data to identify the most physiologically relevant targets. Although these methods are effective, they typically require a large sample size and high-depth sequencing to detect potential miRNA-target pairs, thereby limiting ...